src/mobile/apps/partner_profile/test/client/artworks.test.coffee

```diff
@@ -0,0 +1,48 @@
+benv = require 'benv'
```

Comment:
Not that it matters too much here but if possible new files should be written in typescript (it covers our whole app).